runtime-common/folly/autobuild/defines

```diff
@@ -0,0 +1,14 @@
+PKGNAME=folly
+PKGSEC=libs
+PKGDEP="boost double-conversion fmt gflags glog lz4 openssl snappy xz zlib"
+BUILDDEP="fast-float"
+PKGDES="A C++ library interfacing many other libraries, used by projects made by Facebook"
```

Comment:
Thousand words written, not a single idea came out.

C++17 API component library

Should suffice.